metricbeat/include/list.go

```diff
@@ -85,6 +85,10 @@ import (
 	_ "github.com/elastic/beats/metricbeat/module/system/network"
 	_ "github.com/elastic/beats/metricbeat/module/system/process"
 	_ "github.com/elastic/beats/metricbeat/module/system/socket"
+	_ "github.com/elastic/beats/metricbeat/module/vsphere"
+	_ "github.com/elastic/beats/metricbeat/module/vsphere/datastore_usage"
```

Comment:
Actually thinking about it...No, we don't need. I removed now.
:white_check_mark: